docs/reference/commandline/docker.md

```diff
@@ -1,54 +1,91 @@
----
-title: "Use the Docker command line"
-description: "Docker's CLI command description and usage"
-keywords: "Docker, Docker documentation, CLI, command line, config.json, CLI configuration file"
-redirect_from:
-  - /reference/commandline/cli/
-  - /engine/reference/commandline/engine/
-  - /engine/reference/commandline/engine_activate/
-  - /engine/reference/commandline/engine_check/
-  - /engine/reference/commandline/engine_update/
----
-
-<!-- This file is maintained within the docker/cli GitHub
-     repository at https://github.com/docker/cli/. Make all
-     pull requests against that repo. If you see this file in
-     another repository, consider it read-only there, as it will
-     periodically be overwritten by the definitive file. Pull
-     requests which include edits to this file in other repositories
-     will be rejected.
--->
-
 # docker
 
 To list available commands, either run `docker` with no parameters
 or execute `docker help`:
 
-```console
-$ docker
-Usage: docker [OPTIONS] COMMAND [ARG...]
-       docker [ --help | -v | --version ]
-
-A self-sufficient runtime for containers.
-
-Options:
-      --config string      Location of client config files (default "/root/.docker")
-  -c, --context string     Name of the context to use to connect to the daemon (overrides DOCKER_HOST env var and default context set with "docker context use")
-  -D, --debug              Enable debug mode
-      --help               Print usage
-  -H, --host value         Daemon socket(s) to connect to (default [])
-  -l, --log-level string   Set the logging level ("debug"|"info"|"warn"|"error"|"fatal") (default "info")
-      --tls                Use TLS; implied by --tlsverify
-      --tlscacert string   Trust certs signed only by this CA (default "/root/.docker/ca.pem")
-      --tlscert string     Path to TLS certificate file (default "/root/.docker/cert.pem")
-      --tlskey string      Path to TLS key file (default "/root/.docker/key.pem")
-      --tlsverify          Use TLS and verify the remote
-  -v, --version            Print version information and quit
-
-Commands:
-    attach    Attach to a running container
-    # […]
-```
+<!---MARKER_GEN_START-->
+The base command for the Docker CLI.
```

Comment:
Looks like renaming `cli.md` to `docker.md` doesn't work currently; `docker.yaml` does not include the content; I opened a draft PR to test this PR; https://github.com/docker/docs/pull/16416


Maybe we should have a look at this separately; also to see what we want to split to what page (some of this could go into `index.md`, other bits could go into `../run.md`, and other parts into `docker.md`.